src/Dashboard/Provider.php

```diff
@@ -803,7 +803,9 @@ public static function nbItemByFk(
         if ($fk_item instanceof CommonTreeDropdown) {
             $name = 'completename';
         }
-
+        if (!($item instanceof CommonDevice)) {
```

Comment:
CommonDevice is in the global namespace while Provider is not, so this reference isn't valid.
You can add `use CommonDevice;` to the top of the file with the other use statements to fix the reference.

Reply:
Nevermind, I see you just changed that.